model.go

```diff
@@ -97,12 +97,14 @@ type Feature string
 
 // Available features
 const (
-	SixDice Feature = "six-dice"
+	SixDice      Feature = "six-dice"
+	YahtzeeBonus         = "yahtzee-bonus"
```

Comment:
```suggestion
	YahtzeeBonus Feature = "yahtzee-bonus"
```
iirc if you don't define it explicitly then `YahtzeeBonus` will be a string instead of a `Feature`